dev-python/colorcet/colorcet-2.0.6.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Collection of perceptually uniform colormaps"
+HOMEPAGE="
+	https://pypi.org/project/colorcet
+	https://github.com/holoviz/colorcet"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND=">=dev-python/setuptools-30.3.0[${PYTHON_USEDEP}]"
```

Comment:
We can avoid this, it's set by [default](https://dev.gentoo.org/~mgorny/python-guide/distutils.html#distutils-use-setuptools)

Reply:
Gotcha. <sup>I *thought* that might be the case, but wasn't quite sure.</sup>